### PR TITLE
Inline scales

### PR DESCRIPTION
**Pull request by cassidyjames**
This brings the scales inline with the switches to better associate them and make the layout less lopsided.

Before:

image

After:

image

### CODE REVIEW

File: src/Views/General.vala [hunk 1]
```diff
@@ -82,20 +82,18 @@ public class MouseTouchpad.GeneralView : Gtk.Grid {
...
-        var hold_length_label = new SettingLabel (_("Long-press length:"));
-        hold_length_label.margin_bottom = 18;
+        var hold_length_label = new SettingLabel (_("Length:"));
 
         var hold_scale = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0.5, 2.0, 0.1);
         hold_scale.draw_value = false;
         hold_scale.hexpand = true;
-        hold_scale.margin_bottom = 10;
         hold_scale.width_request = 160;
-        hold_scale.add_mark (1.2, Gtk.PositionType.BOTTOM, null);
```
Comment (peteruithoven) [inline src/Views/General.vala]: Why move the markers to top for these inline scales? This seems inconsistent with other pages?

Comment (cassidyjames) [reply src/Views/General.vala]: It lines the scale up better with the baseline of the text; I'm open to a different way to do that, though.

**Comment by danrabbit** [comment]
So I kinda of wanted to step away from this for a while and see if looking at it with fresh eyes made it feel less busy, but I think I still kind of feel like this makes it a big harder to see what exactly is going on than what we have in master already

File: src/Views/General.vala (hunk 1 above)
Comment (peteruithoven) [reply src/Views/General.vala]: Fair point. Just for reference: 
screenshot from 2019-01-08 01 39 13
screenshot from 2019-01-08 01 38 45

But for such a small difference I wouldn't change the consistency? Maybe the alignment with text of the slider could be improved in general later on? If I for example add both TOP and BOTTOM it's nicely vertically centered with the text.

**Comment by peteruithoven** [comment]
I think, even though it's a bit more crammed, it makes it easier to get an overview of the available options, because it groups the enable/disable with a more specific setting. A bit like we do in the sound indicator:
screenshot from 2019-01-08 01 50 46
And the sound plug:
screenshot from 2019-01-08 01 51 24
Putting them on separate lines gives the impression that there are 2 more "features"?

**Comment by danrabbit** [comment]
These tortoise and hair icons aren't very legible at this size but.. what about icons?

mock

**Comment by cassidyjames** [comment]
@danrabbit yeah I'd be down to investigate that route

**Comment by HansCronau** [comment]
I like the idea of icons, but I'm not sure if animals and a watch would be most intuitive to communicate the concepts of delay/length and cursor speed.

Some thoughts on speed:  
How would you feel about communicating the concept of speed with something that shows a trail? That could still be the hare, but since we're talking cursor speed a mouse pointer might be appropriate. A pointer with a short trail on the left side, and one with a long trail on the right side?

Some thoughts on length:  
Although not icons, how would you feel about mentioning actual milliseconds (ms) for the delay? My reason for considering this is that, whatever the icons, the length slider would remain rather arbitrary without some actual indication of time, be it in ms or even a preview/test for the user to see/experience.

**Pull request merged**